src/stories/Button.stories.ts

```
@@ -0,0 +1,54 @@
+import type { Meta, StoryObj } from '@storybook/nextjs-vite';
+
+import { fn } from 'storybook/test';
+
+import { Button } from './Button';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)